src/assets/js/features/popup/popupSlice.js

```diff
@@ -0,0 +1,38 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const popupSlice = createSlice({
+  name: 'popup',
+  initialState: {
+    loading: false,
+    showSignIn: true,
+    showProposal: false,
+    showSuccess: false,
+    assetDashboardLink: '',
```

Comment:
States like loading, success should ideally be associated to a particular action/side effect.

For example for a sign in action, it should have loading, success, fail states linked to it.